mixes/reducer.js

```diff
@@ -101,18 +104,18 @@ function createReducer (config) {
     ])),
     [deleteMixSuccess]: (state, action) => {
       const nestedMix = action.payload
+      const { id, channel } = nestedMix
 
       const nextRecords = { ...state.records }
-      delete nextRecords[nestedMix.id]
-
-      // TODO: we need to do the inverse of setChannels, setClips. how?
+      delete nextRecords[id]
 
       return loop({
         ...state,
-        dirty: without(state.dirty, nestedMix.id),
+        dirty: without(state.dirty, id),
         records: nextRecords
       }, Effects.batch([
-        Effects.constant(deleteMeta(nestedMix.id)),
+        Effects.constant(unsetChannel(channel)),
```

Comment:
after implementing this i realized we might have some strangeness. here we're calling `unsetChannel` on a channel that's actually nested. iirc you said that in reducers we should be operating on the flat / store representation, NOT the nested / view representation. as a result, `unsetChannel` expects a nested channel, and so does `unsetChannels`.

that being said i dont see a clean way around it. ideally i call `deleteMix(id)`, and then in the reducer, `channelId = state.records[id].channel; unsetChannel(channelId);`. but then to make `saveMix` symmetric, i should call `saveMix(id)` which requires i somehow figure out how to `nestMix`. but this requires i have `channels, clips` which the `saveMix` reducer doesnt have access to!

note to self: if i update to `saveMix(id)`, i should also update to `saveMeta(id)`